tests/integration/pages/experiment_detail.py

```diff
@@ -0,0 +1,34 @@
+"""Representaion of the Experiment Detail Page."""
+
+import random
+import string
+
+from selenium.webdriver.common.by import By
+from pypom import Region
+
+from pages.base import Base
+
+class DetailPage(Base):
+
+    _edit_branches_btn_locator = (By.CSS_SELECTOR,"#branches-edit-btn")
+
+    _page_wait_locator = (
+        By.CSS_SELECTOR,
+        "#overview",
+    )
+
+    def wait_for_page_to_load(self):
+        self.wait.until(
+            lambda _: "Overview"
```

Comment:
Looking at the errors on circleci [here](https://3247-71493337-gh.circle-artifacts.com/0/home/circleci/experimenter/tests/integration/integration-test.html) it looks like this doesn't work anymore. Overview was originally the first page in the create experiment flow. It appears the "Design" page is now loading. This should probably be changed to ```Design```